skbio/diversity/tests/test_driver.py

```diff
@@ -168,20 +168,17 @@ def test_invalid_input_phylogenetic(self):
 
     def test_empty(self):
         # empty vector
-        actual = alpha_diversity('sobs',
-                                 np.array([], dtype=np.int64))
+        actual = alpha_diversity('sobs', np.array([], dtype=np.int64))
         expected = pd.Series([0], dtype=int)
```

Comment:
`expected` should have `dtype=np.int64` to pass on Windows.

This is also true for lines 208 in `test_empty`, line 213 in `test_single_count_vector`, lines 255 and 262 in `test_sobs`, and line 315 in `test_no_ids` in this class.